### DISCUSSION TITLE
Scary stuff 🤔👻🕵🏻‍♂️Anyone else who ever seen THIS???

### DISCUSSION THREAD

**Discussion by MortenSnifSnif** [ish-app/ish#1723] [Q&A]
I opened a file/directory from my /sbin on my iPhone. 
Then this crazy message came???

Check out the video for yourself!
It's super weird. 

Please let me hear if you have any idea or any https://user-images.githubusercontent.com/95707169/154553071-3b2c9727-bc6f-40d2-bbae-cfb1bc22abc4.MOVcomments

**Comment by saagarjha** [comment]
It seems like you forgot to attach a video?

**Reply by kmf** [reply]
[https://user-images.githubusercontent.com/95707169/154553071-3b2c9727-bc6f-40d2-bbae-cfb1bc22abc4.MOV](https://user-images.githubusercontent.com/95707169/154553071-3b2c9727-bc6f-40d2-bbae-cfb1bc22abc4.MOV)
